docs/release-notes/8.5.asciidoc

```diff
@@ -16,6 +23,13 @@
 [[release-notes-8.5.2]]
 === 8.5.2
 
+[discrete]
+[[known-issue-8.5.2]]
+==== Known issues
+* The rule details page and **Edit rule settings** page load indefinitely if you edit a rule that has the `saved_id` property configured. All rule types, except for the custom query rule, are affected.
++
+Upgrading to 8.6 or higher resolves this issue. If you’d prefer to stay on 8.5, use the Update rule API to remove the `saved_id` field from the non-functioning `query`, `eql`, `machine_learning`, `threat_match`, `threshold`, or `new_terms` rule.
```

Comment:
is this section needed to be repeated for every minor version?
I can see that earlier known issues for 8.5 are not repeated for the rest of minor versions

Reply:
Resolving since chatted about this in Slack. Good question though!